routes/movies.routes.js

```diff
@@ -0,0 +1,101 @@
+// starter code in routes/movies.routes.js
+const router = require("express").Router();
+
+// all your routes here
+
+const Movie = require('../models/Movie.model.js');
+
+//Require celebrity model
+const Celebrity = require('../models/Celebrity.model.js')
+
+// Get routes to display all the moviess in the db:
+
+router.get('/movies/create', async (req,res) => {
+    try{
+        let allCelebritiesFromDb = await Celebrity.find()
+        res.render('movies/new-movie.hbs', {celebrities: allCelebritiesFromDb})
+    }
+    catch(error){console.log(error)}
+})
+
+router.post('/movies/create', async (req,res)=>{
+    try{
+        const {title, genre, plot, cast} = req.body;
+        await Movie.create({title, genre, plot, cast})
+        res.redirect('/movies');
+    }
+    catch(error){
+        console.log(error)
+        res.render('movies/new-movie.hbs')
+    }
+})
+
+//GET route to display all the movies in the Db
+router.get('/movies', async (req,res) => {
+    try{
+        let allMoviesFromDb = await Movie.find();
+
+        res.render('movies/movies.hbs', {movies: allMoviesFromDb})
+    }
+    catch(error){
+        console.log(error)
+    }
+})
+
+//GET route for movie details
+router.get('/movies/:movieId', async (req, res) => {
+    try{
+        const {movieId} = req.params;
+
+        let chosenMovie = await Movie.findById(movieId)
+        
+        await chosenMovie.populate('cast');
+
+        res.render('movies/movie-details', {chosenMovie})
```

Comment:
Since you are doing a findById() you receive just one object property, so, in this case, you don't need to send `chosenMovie `inside "{ }".
`res.render("movies/movie-details.hbs", chosenMovie);`